microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md

```diff
@@ -100,6 +98,9 @@ When you run the optimized version of the page, the blue icons move faster.  Why
    ![The results of the profile.](../media/evaluate-performance-performance-capture-results.msft.png)
 
 
```

Comment:
Also change:
Even the optimized version isn't great, because it manipulates the top property of every icon. A better approach is to stick to properties that only affect compositing.
to:
The optimized code uses a different sequence of actions to do less work.  Note that this code could be made even faster by only using properties that only affect compositing, instead of manipulating the top property of every icon.

Reply:
change made